scripts/build/package.js

```diff
@@ -47,10 +47,12 @@ function licenseToObject(license) {
   return license;
 }
 function iconToObject(icon) {
+  const escapedIconTitle = escape(icon.title);
   return util.format(iconObjectTemplate,
-    escape(icon.title),
+    escapedIconTitle,
     escape(icon.slug),
-    escape(icon.svg),
+    escapedIconTitle,
```

Comment:
This one should also have HTML character escaped. A reverse version of [`htmlFriendlyToTitle`](https://github.com/simple-icons/simple-icons/blob/b9cc1788d1efa38de69a17e1c8cb9a86eeed4954/scripts/utils.js#L39-L43) should probably suffice.

Reply:
Good point, done in https://github.com/simple-icons/simple-icons/pull/6505/commits/7b061f62a7cb6751dc30c15033c4bb3d97039978